### PR TITLE
update nested sequence readme.cn.md

### LINKED ISSUE #710: Translation Plan-基于双层序列的文本分类-汉译英
## 任务背景
针对v2版本，目前我们已有很多中文使用文档，但仍有大量的汉译英工作及少量英译汉工作没有完成，我们希望将中英文版本补充完全，提升用户的阅读体验，也有益于国外用户在使用时查阅文档。
此外，我们的模型库Models，仍有大部分英译汉和汉译英的工作，亟待完成。

## 任务描述
1. 需要做什么：
把“基于双层序列的文本分类”**汉译英**

2. 地址
- 中文地址： [Github地址](https://github.com/PaddlePaddle/models/blob/develop/nested_sequence/text_classification/README.md)，[网页地址](http://www.paddlepaddle.org/docs/develop/models/nested_sequence/text_classification/README.html)
- 需要翻译的英文地址，暂无，可在中文版所在github目录下创建英文版md文档

## 如何认领
我们欢迎大家积极认领，内部小伙伴请通过self assign的方式认领，外部小伙伴可通过在本issue下方comment认领。

## 注意事项
上交初稿的截止日期是2018年4月15日
上交后，PaddlePaddle的RD会为大家review
请于review后的5个工作日之内完成修改

## 相关信息
所有需要翻译的文档：
V2使用文档：
https://shimo.im/sheet/PqK407VYm80Ju3dw
Models:
https://shimo.im/sheet/6XTZnRQEUS0uC1Ut

### PR DESCRIPTION
**Pull request by shibing624**
pass ci
fix https://github.com/PaddlePaddle/models/issues/710

### CODE REVIEW

**Comment by shanyi15** [comment]
@peterzhang2029 ,please review this translation at your free time(especially technical terms)
thank you so much for the contribution to PaddlePaddle!

File: nested_sequence/text_classification/README.en.md [hunk 1]
```diff
@@ -3,12 +3,14 @@ The minimum PaddlePaddle version needed for the code sample in this directory is
 ---
 
 # Text classification based on double sequence
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: 标题和副标题除介词外的单词首字母大写。

File: nested_sequence/text_classification/README.en.md [hunk 2]
```diff
@@ -3,12 +3,14 @@ The minimum PaddlePaddle version needed for the code sample in this directory is
...
+
 ## Introduction
 This example will demonstrate how to organize long text(usually paragraphs or chapters) input into a double sequence in PaddlePaddle to complete the task of classifying long text.
+
 ## Model introduction
 We treat a text as a sequence of sentences, and each sentence is a sequence of words.
 
-We first use the convolutional neural network to encode each sentence in the paragraph; then, the expression vector of each sentence goes through the pooled layer to obtain the encoded vector of the paragraph; finally, the encoded vector of the paragraph is used as the classifier input to obtain the final classification result.
+We first use the convolutional neural network to encode each sentence in the paragraph; then, the expression vector of each sentence goes through the pooled layer to obtain the encoded vector of the paragraph; finally, the encoded vector of the paragraph is used as the classifier(the full connection of softmax layer) input to obtain the final classification result.
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: ...then, let the expression vector of each sentence goes through the pooled layer to...

File: nested_sequence/text_classification/README.en.md [hunk 3]
```diff
@@ -32,7 +34,7 @@ The single layer sequence data after disassembly is represented by a CNN network
 
 - **Convolution layer**: convolution in text classification is done on time series. The width of convolution kernel is consistent with the matrix generated by word vector level. After convolution, the result is "feature map". Multiple feature maps can be obtained by using multiple convolutions of different heights. This code uses the convolution kernel of 3 (the red box of Figure 1) and 4 (the blue box of Figure 1) by default.
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: ...the result is a "feature map"

File: nested_sequence/text_classification/README.en.md [hunk 4]
```diff
@@ -32,7 +34,7 @@ The single layer sequence data after disassembly is represented by a CNN network
...
 - **Maximum pool layer**: the maximum pool operation is performed on each feature graph obtained by convolution. Since the feature graph itself is already a vector, the maximum pooling is actually the largest element in the selection of each vector. All the largest elements are spliced together to form a new vector.
-- **Linear projection layer**: the results obtained from different convolutions are spliced into a long vector after the maximum pool level. Then, a linear projection is used to get the representation vectors of corresponding single layer sequences.
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: splices the results from the maximum pool operations into a long vector.

File: nested_sequence/text_classification/README.en.md [hunk 5]
```diff
@@ -57,10 +59,9 @@ def cnn_cov_group(group_input, hidden_size):
 
     return linear_proj
 ```
-
 PaddlePaddle has been encapsulated with a pooled text sequence convolution module: `paddle.networks.sequence_conv_pool`, which can be called directly.
 
-After getting the expression vectors of each sentence, all the sentence vectors are passed through an average pool level, and a vector representation of a sample is obtained. The vector output the final prediction result through a fully connected layer. The code is as follows:
+After getting the expression vectors of each sentence, all the sentence vectors are passed through an average pool level, and a vector representation of a sample is obtained. The vector output the final prediction result through a fully connected layer. The code:
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: The vector outputs the final prediction...

File: nested_sequence/text_classification/README.en.md [hunk 6]
```diff
@@ -99,8 +100,7 @@ class ModelConfig(object):
 
     ...
 ```
-
-Modify the `config.py` to adjust the parameters. For example, by modifying the `use_gpu` parameter to specify whether or not to use GPU for training.
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: Modify 'config.py' to adjust...
For example, we can specify whether or not to use GPU for training by modifying 'use_gpu'.

File: nested_sequence/text_classification/README.en.md [hunk 7]
```diff
@@ -99,8 +100,7 @@ class ModelConfig(object):
...
+Modify the `config.py` to adjust the parameters. For example, modifying the `use_gpu` parameter to specify whether or not to use GPU for training.
 ## Use PaddlePaddle to build built-in data
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: Implement with PaddlePaddle Built-in Data

File: nested_sequence/text_classification/README.en.md [hunk 8]
```diff
@@ -32,7 +34,7 @@ The single layer sequence data after disassembly is represented by a CNN network
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: line 21: Implementation of this network structure is in 'network_conf.py`.

line 23: To process double-level time series, we need to...;
dismantling->dismantles the sentence into words. The input in the step function is a group of non-sequential words.

line 25: （中文版里，这句话应该改为：在词语级别，我们运用 CNN 网络，以词向量为输入，输出学习到的句子的表示；在段落级别，我们通过池化作用，从若干句子的表示中得到段落的表示。）
At the level of words, we obtain the expression of a sentence from word vectors using CNN. At the level of paragraphs, we obtain the expression of a paragraph from the expressions of the sentences in the paragraph through pooling.

File: nested_sequence/text_classification/README.en.md [hunk 9]
```diff
@@ -122,7 +122,7 @@ By default, the prediction script will load a model that trains a pass to test t
 ### Train
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: 107: Execution->Execute
111:  We will run this example with PaddlePaddle's built-in emotional categorization dataset, `imdb` .
113: After training, the model will be stored in the specified directory (the default models directory). At the terminal, execute the following command:
中文版可以改为：训练结束后，模型将被存储在指定目录当中（默认models目录）。在终端执行：
118: 预测脚本将加载、训练一个pass的模型，并用这个模型对 imdb的测试集 进行测试。
The prediction script will load and train a pass model to test `test set of the IMDB`.
 120: Use Custom Data to Train and Predict //标题和副标题最好格式相同。如果用动词的话最好都用动词，如果用动名词的话最好都用动名词。

File: nested_sequence/text_classification/README.en.md [hunk 10]
```diff
@@ -122,7 +122,7 @@ By default, the prediction script will load a model that trains a pass to test t
...
 1.Data structure
 
-The input data format is as follows: one sample of each behavior is separated by `\t` , the first column is a class label, and the second column is the content of the input text. The following are two sample data:
+The input data format is as follows: one sample of each behavior is separated by `\t` , the first column is a class label, and the second column is the content of input text. The following are two sample data:
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: 中文可以改为：每一行为一条样本。以 \t 分隔，第一列是类别标签，第二列是输入文本的内容。
Each line is a sample with class label and text. Class label and text content are seperated by '\t'. The following are two samples:

File: nested_sequence/text_classification/README.en.md [hunk 11]
```diff
@@ -131,7 +131,7 @@ negative        What a terrible movie. I waste so much time.
 
 2.Write the interface of data reading
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: Write the Data Reading Interface

File: nested_sequence/text_classification/README.en.md [hunk 12]
```diff
@@ -131,7 +131,7 @@ negative        What a terrible movie. I waste so much time.
...
 
-A custom data read interface only needs to write a Python generator to implement the logic of **parsing a training sample from the original input text**. The following code fragment is implemented to read the return type of the original data: `paddle.data_type.integer_value_sub_sequence` and `paddle.data_type.integer_value` 
+A custom data read interface only needs to write a Python generator to implement the logic of **parsing a training sample from the original input text**. The following code fragment is implemented to read the return type of the original data: `paddle.data_type.integer_value_sub_sequence` and `paddle.data_type.integer_value`
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: 自定义数据读取接口只需编写一个 Python 生成器，实现从原始输入文本中，解析一条训练样本的逻辑。
//中文版这里最后一句话我无法读懂，可能需要和原作者确认一下
To define a custom data reading interface, we only need to write a Python generator to parse a training sample from the input text.

File: nested_sequence/text_classification/README.en.md (hunk 2 above)
Comment (shibing624) [reply nested_sequence/text_classification/README.en.md]: ok

**Comment by shibing624** [comment]
fix with review result.

File: nested_sequence/text_classification/README.en.md [hunk 13]
```diff
@@ -3,12 +3,14 @@ The minimum PaddlePaddle version needed for the code sample in this directory is
 ---
 
 # Text classification based on double sequence
+
```
Comment (skylarch) [inline nested_sequence/text_classification/README.en.md]: line 1: Running sample code in this directory requires PaddelPaddle v0.11.0 and later. If the PaddlePaddle on your device is lower than this version, please follow the instructions in [installation document](http://www.paddlepaddle.org/docs/develop/documentation/zh/build_and_install/pip_install_cn.html) and make an update.

Comment (shibing624) [reply nested_sequence/text_classification/README.en.md]: ok

**Review by shanyi15 — APPROVED** [review]
LGTM! Thank you very much!

**Pull request merged**